3.6/debian-9/rootfs/opt/bitnami/scripts/mongodb-env.sh

```diff
@@ -112,10 +113,10 @@ export MONGODB_DEFAULT_SYSTEM_LOG_VERBOSITY="0"
 
 # User and database creation settings
 export MONGODB_ROOT_PASSWORD="${MONGODB_ROOT_PASSWORD:-}"
-export MONGODB_USERNAME="${MONGODB_USERNAME:-}"
-export MONGODB_PASSWORD="${MONGODB_PASSWORD:-}"
-export MONGODB_DATABASE="${MONGODB_DATABASE:-}"
-export ALLOW_EMPTY_PASSWORD="${ALLOW_EMPTY_PASSWORD:-no}"
+export MONGODB_USERNAMES="${MONGODB_USERNAMES:-}"
+export MONGODB_PASSWORDS="${MONGODB_PASSWORDS:-}"
+export MONGODB_DATABASES="${MONGODB_DATABASES:-}"
```

Comment:
We can manage the aliases here:
```suggestion
export MONGODB_USERNAMES="${MONGODB_USERNAMES:-"${MONGODB_USERNAME:-}"}"
export MONGODB_PASSWORDS="${MONGODB_PASSWORDS:-"${MONGODB_PASSWORD-}"}"
export MONGODB_DATABASES="${MONGODB_DATABASES:-"${MONGODB_DATABASE:-}"}"
```

Reply:
See my comment around "procedural aliasing" above

Reply:
Replied